plugins/modules/net_tools/gandi_livedns.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019 Gregory Thiemonge <gregory.thiemonge@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
```

Comment:
This needs to be removed.

Reply:
done

I also added integration tests, I can provide the cloud/gandi support in ansible-test as well

Reply:
@gthiemonge do you work for Gandi, or have connections there?

Reply:
no, I'm just a customer of their DNS service

Reply:
Ah, then I don't know what you mean with "I can provide the cloud/gandi support in ansible-test as well". Or do you mean support for a Gandi service emulator?

Reply:
also what about this discussion branch?